create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -233,47 +257,51 @@ void shouldReturnCreatePublicationRequestWithUnconfirmedPublicationContextWhenEv
 
     @Test
     void shouldReturnDefaultPublicationContextWhenEventWithS3UriThatPointsToScopusXmlWithoutPrintIssn()
-            throws IOException {
+        throws IOException {
         var scopusFile = IoUtils.stringFromResources(Path.of("2-s2.0-0000469852.xml"));
         scopusFile = scopusFile.replace("<issn type=\"print\">09604286</issn>", "");
         var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
         var s3Event = createS3Event(uri);
         var createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
         var actualPublicationContext = createPublicationRequest.getEntityDescription().getReference()
-                .getPublicationContext();
+            .getPublicationContext();
         assertThat(actualPublicationContext, instanceOf(UnconfirmedJournal.class));
         var actualJournalName = ((UnconfirmedJournal) actualPublicationContext).getTitle();
         assertThat(actualJournalName, is(HARD_CODED_JOURNAL_NAME_IN_RESOURCE_FILE));
     }
 
     @Test
     void shouldReturnCreatePublicationRequestWithJournalWhenEventWithS3UriThatPointsToScopusXmlWhereSourceTitleIsInNsd()
-            throws IOException {
+        throws IOException {
+
+        URI expectedJournalUri = mockedPublicationChannelsReturnsJournalUri();
+
         var scopusFile = IoUtils.stringFromResources(Path.of("2-s2.0-0000469852.xml"));
-        scopusFile = scopusFile.replace("<xocs:pub-year>1993</xocs:pub-year>", "<xocs:pub-year>2010</xocs:pub-year>");
+        scopusFile = scopusFile.replace("<xocs:pub-year>1993</xocs:pub-year>",
+                                        "<xocs:pub-year>2010</xocs:pub-year>");
         var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
         var s3Event = createS3Event(uri);
         var createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
         var actualPublicationContext = createPublicationRequest.getEntityDescription().getReference()
-                .getPublicationContext();
+            .getPublicationContext();
         assertThat(actualPublicationContext, instanceOf(Journal.class));
         var actualJournalName = ((Journal) actualPublicationContext).getId().toString();
-        assertThat(actualJournalName, is(HARD_CODED_JOURNAL_URI));
+        assertThat(actualJournalName, is(expectedJournalUri.toString()));
```

Comment:
maybe we should aline the to objects before `assertThat(<String>, is(<String>)))` for readability. Or have both checking on object `assertThat(<URI>, is(<URI>))` (that should be fine for URI equals() method) .